### ISSUE TITLE
When the mouse is focused on an input element which is present under the shadowroot then the browser throws an error.

### ISSUE THREAD

**Issue by jetiben** [tinymce/tinymce#6093]
All browsers have this bug.

**Comment by lnewson** [comment]
The details on this issue were pretty scarce, so this took a while to replicate, however I think I did end up finding the exception mentioned. @jetiben can you please confirm this is what's required to reproduce this error?

1. Use TinyMCE within a shadow root
2. An input element must exist as a child of the shadow root and be below the editor
3. When focusing,  you'll get an exception like this:
```
Uncaught TypeError: Cannot read property 'toString' of undefined
    at Object.isEditorUIElement (FocusManager.ts:32)
    at isEditorUIElement (FocusController.ts:23)
    at FocusController.ts:41
    at getParents (DOMUtils.ts:575)
    at Object.getParent (DOMUtils.ts:590)
    at isUIElement (FocusController.ts:39)
    at FocusController.ts:106
    at Object.each (Optional.js:60)
    at Object.documentFocusInHandler (FocusController.ts:103)
    at EventUtils../src/core/main/ts/api/dom/EventUtils.ts.EventUtils.executeHandlers (EventUtils.ts:585)
```

If so, I can confirm this is a bug as TinyMCE is incorrectly trying to lookup the `className` of a document fragment.

**Comment by jetiben** [comment]
Perfectly right.

Another situation, if i direct put the tinymce editor in web components, it seems can not work well. I hope this usage will be supported in the future version, thanks a lot.

**Comment by lnewson** [comment]
Thanks, I'll log this as an internal issue. At this stage shadow dom/web components should largely work, but yeah it's not fully supported unfortunately, but it's something we're aiming for in a future release.

**Comment by gladorange** [comment]
I have the same issue. 
Note, this it's extremely easy to fix it.
I created PR https://github.com/tinymce/tinymce/pull/6133

**Comment by karol-depka** [comment]
Hi. I have the same error.
How about using older version?

E.g. this seems to have stopped the error from appearing:

<script src="https://cdn.tiny.cloud/1/(YOUR_KEY)/tinymce/5.4.2/tinymce.min.js" referrerpolicy="origin"></script>

previously I had `tinymce/5/` in the URL.

**Comment by karol-depka** [comment]
I'm using TinyMCE with Ionic 5 (which is based on WebComponents).

**Comment by lnewson** [comment]
A fix for this has been merged and will be released as part of the upcoming TinyMCE 5.6 release.
